bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -301,16 +311,28 @@
 		<description>The current relative self consumption in %, NULL if no smart meter is connected</description>
 		<state pattern="%.1f %unit%" readOnly="true"></state>
 	</channel-type>
-	<channel-type id="inverter1Power">
+	<channel-type id="inverterPower">
+		<item-type>Number:Power</item-type>
+		<label>Inverter Power</label>
+		<description>Inverter Power</description>
+		<state pattern="%.2f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="inverterSoc">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Inverter State of Charge</label>
+		<description>Inverter State of Charge</description>
+		<state pattern="%.1f %unit%" readOnly="true"></state>
```

Comment:
As item type is Number:Dimensionless, please do not use `%unit%` but rather a real unit. Is it `%` ?